### DISCUSSION TITLE
How to try out V5?

### DISCUSSION THREAD

**Discussion by XVincentX** [lodash/lodash#5930] [General]
I would be curious to try out the V5 version ahead of time, but I am unable to find anything resembling a `fp.ts` file. @jdalton do you have any instructions for who would like to give it a try?

**Comment by maurer2** [comment]
Hello, I believe the lodash-fp module will be removed in v5 according to this comment https://github.com/lodash/lodash/issues/5526#issuecomment-1370458010

Cheers

**Reply by XVincentX** [reply]
Oh that is interesting. I am wondering what the motivations are!

**Comment by Nantris** [comment]
Any update on Lodash v5? It's so widely used it could decrease the size of quite a lot across the web.

**Reply by Chieffo2021** [reply]
@jdalton
